### ISSUE TITLE
PMM-8489 Failed to get topology labels when the target server is mongos

### ISSUE THREAD

**Issue by JamesChenX** [percona/mongodb_exporter#296]
## Steps to Reproduce
1. Set up a mongos server
2. Update the `mongodb.uri` to the mongos and run mongodb_exporter

## Actual
Failed to get topology labels when the target server is mongos, and get the error from logs:
> time="2021-06-14T08:39:16Z" level=error msg="Cannot get topology info: error getting cluster ID: (CommandNotFound) no such cmd: replSetGetConfig: cannot get topology labels"

## Expected
Should work fine with mongos

## Note
It seems the error is caused by: https://github.com/percona/percona-toolkit/blob/49b570ed0ccb49cecd3515d444f3fef44a56d115/src/go/mongolib/util/util.go#L460
because it's not a `ReplicationNotEnabledError` but a `CommandNotFound` error.

## Example
```yaml
  mongo-exporter:
    image: bitnami/mongodb-exporter:0.20.5
    command:
      - '--disable.replicasetstatus'
    depends_on:
      - mongodb-router
    environment:
      - MONGODB_URI=mongodb://mongodb-router:27017
    restart: always
    ports:
      - 9216:9216

  mongodb-router:
    image: "bitnami/mongodb-sharded:4.4.4"
    environment:
      - MONGODB_ADVERTISED_HOSTNAME=mongodb-router
      - MONGODB_SHARDING_MODE=mongos
      - MONGODB_CFG_PRIMARY_HOST=mongodb-cfg
      - MONGODB_CFG_REPLICA_SET_NAME=cfgreplicaset
      - MONGODB_REPLICA_SET_KEY=replicasetkey123
    ports:
      - "27017:27017"
```

**Comment by percona-csalguero** [comment]
Created https://jira.percona.com/browse/PMM-8489 to track this issue

**Comment by percona-csalguero** [comment]
PR: https://github.com/percona/mongodb_exporter/pull/325/files

**Comment by mjseid** [comment]
@percona-csalguero I'm seeing this as well with a sharded cluster pointing to mongos's.  Since your PR is not merged/released yet, I tried to do a custom build from your branch and it is still giving the same error for me.  Not sure if I did the build correctly

```
git clone https://github.com/percona/mongodb_exporter.git
cd mongodb_exporter/
git checkout PMM-8489_topology_labels_on_mongos
make release
```

I'm running the resulting binary like this:
```
mongodb_exporter --disable.replicasetstatus --web.listen-address=":9217" --compatible-mode --mongodb.direct-connect=false --mongodb.uri="mongodb://user:password@mongos-0.my-svc.mongodb-dev.svc.cluster.local:27017,mongos-1.my-svc.mongodb-dev.svc.cluster.local:27017,mongos-2.my-svc.mongodb-dev.svc.cluster.local:27017/admin?tls=true&tlsAllowInvalidCertificates=true&tlsCAFile=/opt/bitnami/custom/ca-pem"
```

result is 
```
INFO[0000] Starting HTTP server for http://:9217/metrics ...  source="server.go:140"
ERRO[0030] Cannot get topology info: error getting cluster ID: (CommandNotFound) no such cmd: replSetGetConfig: cannot get topology labels
```

**Comment by percona-csalguero** [comment]
Hello, 

MongoDB exporter can't handle multiple servers in the connection list. 
Could you specify only one and try again please?

Thanks

**Comment by mjseid** [comment]
Ah, I guess the issue must have been on  my end.  I got the 0.11.2 version working yesterday but I tried the released 0.20.7 version today and its working now.  

Very sorry for the confusion, but thank you for checking with me.

Also just to note, providing a string of 3 mongos servers in the URI does work for me for both the 0.11.2 and the 0.20.7 versions
